src/main/g8/src/main/scala/$package$/Lambda.scala

```diff
@@ -0,0 +1,50 @@
+package $package$
+
+import com.amazonaws.services.lambda.runtime.Context
+
+import scala.beans.BeanProperty
+
+/**
+  * This is compatible with aws' lambda JSON to POJO conversion.
+  * You can test your lambda by sending it the following payload:
+  * {"name": "Bob"}
+  */
+class LambdaInput() {
+  @BeanProperty var name: String = _
```

Comment:
I'm personally not a fan of the `scala.beans.BeanProperty`. It makes something simple appear far more complicated to those unfamiliar.